### ISSUE TITLE
Keyboard Macro Cant Navigate UAC Prompt.

### ISSUE THREAD

**Issue by brycebba** [rivafarabi/deckboard#299]
I am trying to write a simple Advanced Keyboard Macro to do the following Win + X, A, Left, Enter. This should launch powershell with Administrative privileges. The problem is that my macro stops doing anything when the UAC prompt comes up so the Left, Enter commands to say Yes, open powershell as Admin fails to execute or fails to execute on the focused UAC. I even went as far as running Deckboard in Administrator Mode to see if it needed elevated priveleges to interact with the UAC prompt and that didnt have any effect. Below is an example of the script. The DELAY 1000 is to make sure the UAC has time to open before hitting DOWN LEFT... and so on. Thoughts?

![image](https://user-images.githubusercontent.com/25572008/68056891-86a6cc80-fcb1-11e9-9905-b6a161f6085f.png)

**Comment by Yurlyn** [comment]
Isn't that by design so a program can not control the cursor for the UAC? I mean, it could result in malicious programs doing that if Microsoft allowed it. Maybe you could make a workaround using AutoHotKey to see if that's the case or not.

**Comment by brycebba** [comment]
Yes it is by design however, elevated programs can access the UAC by using directions. I actually tried AHK as well and it wouldnt control the LEFT and ENTER key with the UAC up. Ultimately I will probably disable UAC on my work machine and just deal with it disabled so that I can have the advanced functionality of elevated programs like Visual Studio and Powershell
